graphdatascience/session/dedicated_sessions.py

```diff
@@ -90,8 +90,8 @@ def get_or_create(
 
         session_connection = DbmsConnectionInfo(
             uri=session_details.bolt_connection_url(),
-            username=self._aura_api._credentials[0],
-            password=self._aura_api._credentials[1],
+            username="",
+            password=self._aura_api._auth._auth_token(),
```

Comment:
what if the token expires? Inside the ArrowQueryRunner we wont be able to refresh the Aura API token?

Reply:
You're right, that needs to be handled. I guess it is a little bit more involved than initially thought :)